threadedcomments/south_migrations/0001_initial.py

```diff
@@ -3,21 +3,26 @@
 from south.db import db
 from south.v2 import SchemaMigration
 from django.db import models
+from django.db import connection
 
 
 class Migration(SchemaMigration):
 
     def forwards(self, orm):
+        is_index = connection.vendor != 'mysql'
         # Adding model 'ThreadedComment'
         db.create_table('threadedcomments_comment', (
             ('comment_ptr', self.gf('django.db.models.fields.related.OneToOneField')(to=orm['comments.Comment'], unique=True, primary_key=True)),
             ('title', self.gf('django.db.models.fields.TextField')(blank=True)),
             ('parent', self.gf('django.db.models.fields.related.ForeignKey')(default=None, related_name='children', null=True, blank=True, to=orm['threadedcomments.ThreadedComment'])),
             ('last_child', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['threadedcomments.ThreadedComment'], null=True, blank=True)),
-            ('tree_path', self.gf('django.db.models.fields.CharField')(db_index=True, max_length=500)),
+            ('tree_path', self.gf('django.db.models.fields.CharField')(db_index=is_index, max_length=500)),
```

Comment:
Why have this option while you have `db_index=False` in other places?
That seems to contradict itself.